src/util/is-valid-url-test.ts

```diff
@@ -0,0 +1,14 @@
+import { isValidUrl } from "utils/is-valid-url";
+
+describe("Is valid url", () => {
+	it("should return true if valid url is provided", async () => {
+		expect(isValidUrl("http://myvalidurl.com")).toBeTruthy();
+		expect(isValidUrl("https://myvalidurl.net")).toBeTruthy();
+	});
+
+	it("should return false if invalid url is provided", async () => {
+		expect(isValidUrl("noprotocolortopleveldomain")).toBeFalsy();
+		expect(isValidUrl("https://www.ihavenottopleveldomain")).toBeFalsy();
```

Comment:
the `www.` will cause the `hostname.split(".").length < 2)` to be tru :(

Reply:
Realised that. Copied the logic from what was implemented on the frontend but realised it doesn't work.